pkg/sdk/notification_integrations_dto_builders_gen.go

```diff
@@ -0,0 +1,289 @@
+	s := CreateNotificationIntegrationRequest{}
+	s.name = name
+	s.Enabled = Enabled
+	return &s
+}
+
+func (s *CreateNotificationIntegrationRequest) WithOrReplace(OrReplace *bool) *CreateNotificationIntegrationRequest {
+	s.OrReplace = OrReplace
+	return s
+}
+
+func (s *CreateNotificationIntegrationRequest) WithIfNotExists(IfNotExists *bool) *CreateNotificationIntegrationRequest {
+	s.IfNotExists = IfNotExists
+	return s
+}
+
+func (s *CreateNotificationIntegrationRequest) WithAutomatedDataLoadsParams(AutomatedDataLoadsParams *AutomatedDataLoadsParamsRequest) *CreateNotificationIntegrationRequest {
+	s.AutomatedDataLoadsParams = AutomatedDataLoadsParams
+	return s
+}
+
+func (s *CreateNotificationIntegrationRequest) WithPushNotificationParams(PushNotificationParams *PushNotificationParamsRequest) *CreateNotificationIntegrationRequest {
+	s.PushNotificationParams = PushNotificationParams
+	return s
+}
+
+func (s *CreateNotificationIntegrationRequest) WithEmailParams(EmailParams *EmailParamsRequest) *CreateNotificationIntegrationRequest {
+	s.EmailParams = EmailParams
+	return s
+}
+
+func (s *CreateNotificationIntegrationRequest) WithComment(Comment *string) *CreateNotificationIntegrationRequest {
+	s.Comment = Comment
+	return s
+}
+
+func NewAutomatedDataLoadsParamsRequest() *AutomatedDataLoadsParamsRequest {
+	return &AutomatedDataLoadsParamsRequest{}
+}
+
+func (s *AutomatedDataLoadsParamsRequest) WithGoogleAutomatedDataLoad(GoogleAutomatedDataLoad *GoogleAutomatedDataLoadRequest) *AutomatedDataLoadsParamsRequest {
+	s.GoogleAutomatedDataLoad = GoogleAutomatedDataLoad
+	return s
+}
+
+func (s *AutomatedDataLoadsParamsRequest) WithAzureAutomatedDataLoad(AzureAutomatedDataLoad *AzureAutomatedDataLoadRequest) *AutomatedDataLoadsParamsRequest {
+	s.AzureAutomatedDataLoad = AzureAutomatedDataLoad
+	return s
+}
+
+func NewGoogleAutomatedDataLoadRequest(
+	GcpPubsubSubscriptionName string,
+) *GoogleAutomatedDataLoadRequest {
+	s := GoogleAutomatedDataLoadRequest{}
+	s.GcpPubsubSubscriptionName = GcpPubsubSubscriptionName
+	return &s
+}
+
+func NewAzureAutomatedDataLoadRequest(
+	AzureStorageQueuePrimaryUri string,
+	AzureTenantId string,
+) *AzureAutomatedDataLoadRequest {
+	s := AzureAutomatedDataLoadRequest{}
+	s.AzureStorageQueuePrimaryUri = AzureStorageQueuePrimaryUri
+	s.AzureTenantId = AzureTenantId
+	return &s
+}
+
+func NewPushNotificationParamsRequest() *PushNotificationParamsRequest {
+	return &PushNotificationParamsRequest{}
+}
+
+func (s *PushNotificationParamsRequest) WithAmazonPush(AmazonPush *AmazonPushRequest) *PushNotificationParamsRequest {
+	s.AmazonPush = AmazonPush
+	return s
+}
+
+func (s *PushNotificationParamsRequest) WithGooglePush(GooglePush *GooglePushRequest) *PushNotificationParamsRequest {
+	s.GooglePush = GooglePush
+	return s
+}
+
+func (s *PushNotificationParamsRequest) WithAzurePush(AzurePush *AzurePushRequest) *PushNotificationParamsRequest {
+	s.AzurePush = AzurePush
+	return s
+}
+
+func NewAmazonPushRequest(
```

Comment:
should this be called `NewAmazonCloudParamsPush` or alternatively `NewAmazonParamsPush`?